[BE-2374]: pinned latest specific version of bundler that fits with de…

boztopuz opened the pull request:
…fault ruby version

boztopuz committed:
[BE-237]: pinned latest specific version of bundler that fits with default ruby version

boztopuz committed:
Fix: changed bundle version with variable

csonuryilmaz commented on main.rb:
```diff
@@ -33,12 +34,12 @@ def runCommand(command)
 	if OS.mac?
 		if `which rbenv`.empty?
-			runCommand("sudo gem install bundler")
+			runCommand("sudo gem install bundler #{bundler_version}")
```
@boztopuz All our runner environments have `rbenv` installed. So, practically, this branch in the **if** is unreachable. :bulb: 

When we remove it, `OS.mac?` check also becomes obsolete since the **if** and **else** branches have same content.

Finally we can have one line gem install here. I will refactor it. :raising_hand_man:

csonuryilmaz committed:
Refactoring

https://github.com/appcircleio/appcircle-fastlane-component/pull/1/files#r1430040627
